frontend/ui/widget/menu.lua

```diff
@@ -89,18 +87,8 @@ end
 Widget that displays an item for menu
 --]]
 local MenuItem = InputContainer:extend{
-    text = nil,
-    bidi_wrap_func = nil,
-    show_parent = nil,
-    detail = nil,
-    font = "cfont",
-    font_size = 24,
+    font = "smallinfofont",
     infont = "infont",
-    infont_size = 18,
-    dimen = nil,
-    shortcut = nil,
-    shortcut_style = "square",
-    _underline_container = nil,
```

Comment:
I usually like having this kind of lists, as documentation.
But I guess you removed them because 1) MenuItem is private, so no need to document it - and 2) it's only used once, so the site where we do MenuItem:new{...} is where all these are document. Right?

Reply:
https://github.com/koreader/koreader/blob/a1928ee3358a1f7cc28e3ef3858707a86138795f/frontend/ui/widget/menu.lua#L1052-L1079
"All or nothing at all". Which one deserves mentioning?

Reply:
"Right" would have been enough :)
Fine with me. It's fine in `local item_tmp = MenuItem:new{ `, where the type of the items can be guessed.